src/bun.js/bindings/bindings.cpp

```diff
@@ -4048,37 +4072,79 @@ class V8StackTraceIterator {
             return false;
         }
 
-        auto firstColon = line.find(':', openingParenthese + 1);
+        auto lineInner = StringView_slice(line, openingParenthese + 1, closingParenthese);
+        auto colonCount = StringView_count(lineInner, ':', 0);
```

Comment:
This is going to make it a _lot_ slower. Do we really need an exact count? Or can we get away with finding the last 2 colons?